modules/eclickadsBidAdapter.js

```diff
@@ -0,0 +1,103 @@
+import { config } from '../src/config.js';
+import { NATIVE } from '../src/mediaTypes.js';
+import { convertOrtbRequestToProprietaryNative } from '../src/native.js';
+import { getWindowTop, getWindowSelf } from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+
+// ***** ECLICKADS ADAPTER *****
+export const BIDDER_CODE = 'eclickads';
+const DEFAULT_CURRENCY = ['USD'];
+const DEFAULT_TTL = 1000;
+export const ENDPOINT = 'https://g.eclick.vn/rtb_hb_request?fosp_uid=';
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [NATIVE],
+  isBidRequestValid: (bid) => {
+    return !!bid && !!bid.params && !!bid.bidder && !!bid.params.zid;
+  },
+  buildRequests: (validBidRequests = [], bidderRequest) => {
+    // FORMAT REQUEST TO BID SERVER
+    validBidRequests = convertOrtbRequestToProprietaryNative(validBidRequests);
+
+    // ***** GET FIRST PARTY DATA CONFIG (SPECIFICLY FOR ECLICKADS) *****
+    const siteConfigFPD = config.getConfig('site') || {};
```

Comment:
Get these from the request

Reply:
Changed to take these params from request

Reply:
Thank you